src/rgw/rgw_rados.cc

```diff
@@ -3764,6 +3764,22 @@ int RGWRados::init_rados()
     }
   }
 
+  {
```

Comment:
@liewegas strongly approve #1;  as regards #2, if some invocations of radosgw-admin do register, will it be confusing when those drop off (finish or are stopped)?